modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/service/dependencies/readonly/ddm-image.ftl

```diff
@@ -8,6 +8,7 @@
 
 		alt = fileJSONObject.getString("alt")
 		src = fileJSONObject.getString("data")
+		fileEntry = getFileEntry(fileJSONObject)
```

Comment:
Hi @jgarcialr ,
Would it be possible to, instead of having two `if` statements below, do something like:

```
<#assign
       alt = fileJSONObject.getString("alt")
       src = fileJSONObject.getString("data")
/>

<#if validator.isNull(src)>
	<#assign src = getFileEntryURL(getFileEntry(fileJSONObject)) />
</#if>